standalone/pherry/src/lib.rs

```diff
@@ -756,7 +756,7 @@ async fn register_worker(
         .tx()
         .phala_registry()
         .register_worker(pruntime_info, attestation)
-        .sign_and_submit_then_watch(signer)
+        .sign_and_submit_then_watch(signer, Default::default())
```

Comment:
Not sure if it implies the longevity is "immutable". If so, theoretically we should still set it to a proper value like what we did for `sync_mq_egress`.

Reply:
set